wolfcrypt/src/asn.c

```diff
@@ -385,7 +385,7 @@ time_t ksdk_time(time_t* timer)
     if (timer == NULL)
         timer = &localTime;
 
-    *timer = (PIT_DRV_ReadTimerUs(PIT_INSTANCE, PIT_CHANNEL)) / 1000000;
+    *timer = COUNT_TO_USEC(PIT_GetCurrentTimerCount(PIT_INSTANCE, PIT_CHANNEL), SystemCoreClock) / 1000000;
```

Comment:
This line may be an issue, as the original code had no dependency on a specific clock. However, the old method is removed in KSDK 2.0 in favor of the COUNT_TO_USEC macro, so we have to include a clock here.